docs/pages/api-reference/liveblocks-yjs.mdx

```diff
@@ -121,6 +121,19 @@ yProvider.on("sync", (sync: boolean) => /* ... */);
 yProvider.on("synced", (sync: boolean) => /* ... */);
 ```
 
+### LiveblocksProvider.on("subdocs") [#LiveblocksProvider.on.subdocs]
+
+Add an event listener for the `subdocs` event. The `subdocs` event is triggered
```

Comment:
I know `Y.Doc` has a listener to the `subdocs` event but wasn't aware we also emitted them from our `LiveblocksProvider`. @jrowny Could you confirm if this is true? 

```ts
doc.on('subdocs', ({ added, removed, loaded }) => {
    ...
})
```

Reply:
This is an error! I misread the Yjs docs 👍